SwrveSDK/src/main/assets/youtubeapi.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<body style = "padding: 0; margin: 0;">
+<div style = "width: 100%; height: 100%;" id="player"></div>
+
+<script>
```

Comment:
We could. BUT, There are a lot of side effects to think about in the case. 
Storage, Extra Loading times, Support for the file would need to be maintained across all SDK's

This file is really just a much nicer way of generating HTML and not needing to make big long Strings in the java file.